File: src/main/kotlin/com/example/testfordatabase/swagger/api/UpdateUserData.java
package com.example.testfordatabase.swagger.api;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.swagger.annotations.ApiModelProperty;

import javax.validation.constraints.*;
import java.util.Objects;

/**
 * UpdateUserData
 */
@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2023-03-19T14:53:52.454350+01:00[Europe/Berlin]")
public class UpdateUserData   {
  @JsonProperty("email")
  private String email;

  @JsonProperty("token")
  private String token;

  @JsonProperty("username")
  private String username;

  @JsonProperty("bio")
  private String bio;

  @JsonProperty("image")
  private String image;

  public UpdateUserData email(String email) {
    this.email = email;
    return this;
  }

  /**
   * Get email
   * @return email
  */
  @ApiModelProperty(value = "")


  public String getEmail() {
    return email;
  }

  public void setEmail(String email) {
    this.email = email;
  }

  public UpdateUserData token(String token) {
    this.token = token;
    return this;
  }

  /**
   * Get token
   * @return token
  */
  @ApiModelProperty(value = "")


  public String getToken() {
    return token;
  }

  public void setToken(String token) {
    this.token = token;
  }

  public UpdateUserData username(String username) {
    this.username = username;
    return this;
  }

  /**
   * Get username
   * @return username
  */
  @ApiModelProperty(value = "")


  public String getUsername() {
    return username;
  }

  public void setUsername(String username) {
    this.username = username;
  }

  public UpdateUserData bio(String bio) {
    this.bio = bio;
    return this;
  }

  /**
   * Get bio
   * @return bio
  */
  @ApiModelProperty(value = "")


  public String getBio() {
    return bio;
  }

  public void setBio(String bio) {
    this.bio = bio;
  }

  public UpdateUserData image(String image) {
    this.image = image;
    return this;
  }

  /**
   * Get image
   * @return image
  */
  @ApiModelProperty(value = "")


  public String getImage() {
    return image;
  }

  public void setImage(String image) {
    this.image = image;
  }


  @Override
  public boolean equals(Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    UpdateUserData updateUser = (UpdateUserData) o;
    return Objects.equals(this.email, updateUser.email) &&
        Objects.equals(this.token, updateUser.token) &&
        Objects.equals(this.username, updateUser.username) &&
        Objects.equals(this.bio, updateUser.bio) &&
        Objects.equals(this.image, updateUser.image);
  }

  @Override
  public int hashCode() {
    return Objects.hash(email, token, username, bio, image);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class UpdateUserData {\n");
    
    sb.append("    email: ").append(toIndentedString(email)).append("\n");
    sb.append("    token: ").append(toIndentedString(token)).append("\n");
    sb.append("    username: ").append(toIndentedString(username)).append("\n");
    sb.append("    bio: ").append(toIndentedString(bio)).append("\n");
    sb.append("    image: ").append(toIndentedString(image)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: auth/src/main/kotlin/com/example/auth/swagger/api/UpdateUserData.kt
package com.example.testfordatabase.swagger.api

import com.fasterxml.jackson.annotation.JsonProperty
import io.swagger.annotations.ApiModelProperty
import java.util.*
import javax.annotation.Generated

/**
 * UpdateUserData
 */
@Generated(
    value = ["org.openapitools.codegen.languages.SpringCodegen"],
    date = "2023-03-19T14:53:52.454350+01:00[Europe/Berlin]"
)
data class UpdateUserData(
    @get:ApiModelProperty(value = "")
    @JsonProperty("email")
    var email: String? = null,

    @get:ApiModelProperty(value = "")
    @JsonProperty("token")
    var token: String? = null,

    @get:ApiModelProperty(value = "")
    @JsonProperty("username")
    var username: String? = null,

    @get:ApiModelProperty(value = "")
    @JsonProperty("bio")
    var bio: String? = null,

    @get:ApiModelProperty(value = "")
    @JsonProperty("image")
    var image: String? = null
) {


    fun email(email: String?): UpdateUserData {
        this.email = email
        return this
    }

    fun token(token: String?): UpdateUserData {
        this.token = token
        return this
    }

    fun username(username: String?): UpdateUserData {
        this.username = username
        return this
    }

    fun bio(bio: String?): UpdateUserData {
        this.bio = bio
        return this
    }

    fun image(image: String?): UpdateUserData {
        this.image = image
        return this
    }


    override fun toString(): String {
        val sb = StringBuilder()
        sb.append("class UpdateUserData {\n")
        sb.append("    email: ").append(toIndentedString(email)).append("\n")
        sb.append("    token: ").append(toIndentedString(token)).append("\n")
        sb.append("    username: ").append(toIndentedString(username)).append("\n")
        sb.append("    bio: ").append(toIndentedString(bio)).append("\n")
        sb.append("    image: ").append(toIndentedString(image)).append("\n")
        sb.append("}")
        return sb.toString()
    }

    /**
     * Convert the given object to string with each line indented by 4 spaces
     * (except the first line).
     */
    private fun toIndentedString(o: Any?): String {
        return o?.toString()?.replace("\n", "\n    ") ?: "null"
    }
}